projects/reservoir/fv3/era_process.py

```diff
@@ -0,0 +1,180 @@
+import xarray as xr
+import glob
+import numpy as np
+import vcm
+import vcm.catalog
+from vcm.catalog import catalog
+import xarray as xr
+import netCDF4 as nc
+import glob
+import cftime
+import datetime
+import os
+import argparse
+import os
+
+FREGRID_EXAMPLE_SOURCE_DATA='gs://vcm-ml-raw/2020-11-12-gridspec-orography-and-mosaic-data/C48/*.nc'
+
+var_id_mapping={'sst':'sst','temp2m': 't2m','u_wind':'u10','v_wind':'v10'}
+
+
+
+def add_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--path",help="path to raw data")
```

Comment:
I'd recommend calling this `input_path` to be more specific. Could you add more detail to the help string to tell the user that this path is expected to contain the data in `path+ var +'/downloaded`?